<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.147727 rgba(0, 135, 205, 255), stop:1 rgba(255, 255, 255, 255));}</string>
   </property>
   <widget class="QCommandLinkButton" name="login_member_bck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>185</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Back page</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_member_make_btn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
font: 20pt &quot;Times New Roman&quot;;
background-color: rgb(0, 0, 0);
color:rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Do it</string>
    </property>
   </widget>
   <widget class="QComboBox" name="login_member_comboBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>320</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="welcome_label_member">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>821</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Welcome&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>1011</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoScrollMargin">
     <number>16</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>39</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>84</number>
    </attribute>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Surname</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Doctor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Task1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Task2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Monday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Thursday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Friday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Saturday</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sunday</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>190</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Weekly situation</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
